src/io/DumpH5MD.cpp

```diff
@@ -0,0 +1,262 @@
+/*
+  Copyright (c) 2015
+      Jakub Krajniak (jkrajniak at gmail.com)
+  Copyright (C) 2012,2013
```

Comment:
I think these lines can be dropped as the reader was added in 2015 and didn't existed back in 2008-2013.